typescript/src/schema/proto.ts

```diff
@@ -393,22 +392,26 @@ function getMethodConfig(
 ): protos.grpc.service_config.MethodConfig {
   let exactMatch: protos.grpc.service_config.IMethodConfig | undefined;
   let serviceMatch: protos.grpc.service_config.IMethodConfig | undefined;
-  for (const config of grpcServiceConfig.methodConfig) {
-    if (!config.name) {
-      continue;
-    }
-    for (const name of config.name) {
-      if (name.service === serviceName && !name.method) {
-        serviceMatch = config;
+  if (Array.isArray(grpcServiceConfig.methodConfig)) {
```

Comment:
Should there be an error or alert if methodConfig is not an array?

Reply:
Yes, this is a good idea to let the user know that their gRPC service config is broken. Added a warning. (failure might be too much I guess?)